### DISCUSSION TITLE
ESPResSo++ weekly meeting (4.00 pm, 15. Dec)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#474] [Team Posts]
The meeting ID has been reactivated now
______
Join Zoom Meeting
https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09

Meeting-ID: 926 0512 4006
Kenncode: 120525

**Comment by mrampp** [comment]
I cannot connect, zoom says "meeting has been ended by the host"

On Thu, Dec 15 2022, Zhen-Hao Xu wrote:
> The meeting ID has been reactivated now
> ______
> Join Zoom Meeting
> https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
> 
> Meeting-ID: 926 0512 4006
> Kenncode: 120525
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/orgs/espressopp/teams/discussion/discussions/20
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
best regards / schoene Gruesse,
Markus Rampp

--  
---------------------------------------------------------
Dr. Markus Rampp
Max Planck Computing and Data Facility (MPCDF)
Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
Phone: +49-89-3299-1324           FAX: +49-89-3299-1301
E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
---------------------------------------------------------

**Comment by xzhh** [comment]
this morning i checked it was reopened. @stuehn have you closed it ?

**Comment by junghans** [comment]
Same issue as Markus

**Comment by stuehn** [comment]
Dear all,

I have created a new zoom-link for our meeting, see below

Best,
Torsten


Thema: ESPResSo++ Developer Meeting
Uhrzeit: 15.Dez. 2022 04:00 PM Amsterdam, Berlin, Rom, Stockholm, Wien

Zoom-Meeting beitreten
https://zoom.us/j/99468001371?pwd=RWVGUmRIQXlWdUFMRnNnM0loZnpyQT09

Meeting-ID: 994 6800 1371
Kenncode: 537959
Schnelleinwahl mobil
+16892781000,,99468001371#,,,,*537959# Vereinigte Staaten
+17193594580,,99468001371#,,,,*537959# Vereinigte Staaten

Einwahl nach aktuellem Standort
         +1 689 278 1000 Vereinigte Staaten
         +1 719 359 4580 Vereinigte Staaten
         +1 929 205 6099 Vereinigte Staaten (New York)
         +1 253 205 0468 Vereinigte Staaten
         +1 253 215 8782 Vereinigte Staaten (Tacoma)
         +1 301 715 8592 Vereinigte Staaten (Washington DC)
         +1 305 224 1968 Vereinigte Staaten
         +1 309 205 3325 Vereinigte Staaten
         +1 312 626 6799 Vereinigte Staaten (Chicago)
         +1 346 248 7799 Vereinigte Staaten (Houston)
         +1 360 209 5623 Vereinigte Staaten
         +1 386 347 5053 Vereinigte Staaten
         +1 507 473 4847 Vereinigte Staaten
         +1 564 217 2000 Vereinigte Staaten
         +1 646 931 3860 Vereinigte Staaten
         +1 669 444 9171 Vereinigte Staaten
         +1 669 900 6833 Vereinigte Staaten (San Jose)
Meeting-ID: 994 6800 1371
Kenncode: 537959
Ortseinwahl suchen: https://zoom.us/u/acyoXwblFf


On 12/15/22 16:01, mrampp wrote:
> I cannot connect, zoom says "meeting has been ended by the host"
> 
> On Thu, Dec 15 2022, Zhen-Hao Xu wrote:
>  > The meeting ID has been reactivated now
>  > ______
>  > Join Zoom Meeting
>  > https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
>  >
>  > Meeting-ID: 926 0512 4006
>  > Kenncode: 120525
>  >
>  > --
>  > Reply to this email directly or view it on GitHub:
>  > https://github.com/orgs/espressopp/teams/discussion/discussions/20
>  > You are receiving this because you are subscribed to this thread.
>  >
>  > Message ID: ***@***.***>
> 
> -- 
> best regards / schoene Gruesse,
> Markus Rampp
> 
> -- 
> ---------------------------------------------------------
> Dr. Markus Rampp
> Max Planck Computing and Data Facility (MPCDF)
> Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
> Phone: +49-89-3299-1324 FAX: +49-89-3299-1301
> E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
> ---------------------------------------------------------
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/orgs/espressopp/teams/discussion/discussions/20/comments/1>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADZSLBMBUTLQMHJ3MBJ2XDWNMXD5ANCNFSM6AAAAAAS7XKQXU>.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 

-- 
Max-Planck-Institut für Polymerforschung
Dr. Torsten Stuehn (Head of Scientific IT)
Ackermannweg 10
55128 Mainz
Tel +49-(0)6131-379268
